docs/configuration.rst

```diff
@@ -135,6 +135,18 @@ The log file will not be rotated automatically as this is problematic due to Ral
             notifempty              # don't attempt to rotate empty ones
     }
 
+If you are benchmarking source builds of Elasticsearch, Rally will write the log output of the build to ``~/.rally/logs/build.log``. Similarly to the configuration for ``rally.log``, you can use the following ``logrotate`` configuration as a starting point::
+
+    /home/user/.rally/logs/build.log {
+            daily                   # rotate daily
```

Comment:
In-line comments don't work. They should be in a separate line.

(From [man logrotate](https://linux.die.net/man/8/logrotate)

> Note that comments may appear anywhere in the config file as long as the first non-whitespace character on the line is a #.
)

We need to update also the example for `rally.log`.

Reply:
addressed in https://github.com/elastic/rally/pull/818/commits/dd68f38f65058f1e9029ce8ade1e0d7071075de6.